action/index.js

```diff
@@ -1,12 +1,20 @@
+import {
+  ensureTaskContext,
+  globalContext,
+  isContext,
+  withContext
+} from '../context/index.js'
 import { startTask } from '../task/index.js'
 
 export let lastAction = Symbol()
 export let actionId = Symbol()
 
-let uid = 0
+export let doAction = ($$store, actionName, cb, args, ctx = globalContext) => {
```

Comment:
Maybe context API fort actions will look like:

```js
renamePost(withContext($post), 'New title')
```

Reply:
I prefer to have single API for contexts and using latest argument can be tricky

Reply:
@ai Yeah, that can be done, sure. Only thing is that won't change the broken contract with the last argument for `action`. I couldn't come up with a better approach, tbh. Any ideas?